include/deal.II/base/quadrature_lib.h

```diff
@@ -47,6 +47,61 @@ class QGauss : public Quadrature<dim>
   QGauss(const unsigned int n);
 };
 
+/**
+ * The Gauss-Radau family of quadrature rules for numerical integration.
+ *
+ * This modification of the Gauss quadrature uses one of the two interval end
+ * points as well. Being exact for polynomials of degree <i>2n-2</i>, this
+ * formula is suboptimal by one degrees.
+ *
+ * This formula is often used in the context of discontinuous Galerkin
+ * discretizations of ODEs and the temporal part of PDEs.
+ *
+ * The quadrature points are the left interval end point plus the roots of the
+ * polynomial <i>(P<sub>n-1</sub>(x)+P<sub>n</sub>(x))/(1+x)</i>
```

Comment:
This works, but you can also use $latex-style$ formulas if you think that would be easier to read.

Reply:
Yes it is easier to read. Changed it, thanks!